.github/workflows/cleanup.yaml

```diff
@@ -6,7 +6,6 @@ jobs:
 
   snowflake-delete:
     name: Delete Snowflake development environment
-    if: ${{ github.ref_name != 'main' && github.ref_name != 'staging' }}
```

Comment:
Why was this removed?

Reply:
We want to use `github.event.ref` instead.

Reply:
sorry i meant the entire conditional but i see it back now